diff -ru gmsl-1.2.3.orig/__gmsl gmsl-1.2.3/__gmsl
--- gmsl-1.2.3.orig/__gmsl	2025-03-23 14:24:08.630638153 +0100
+++ gmsl-1.2.3/__gmsl	2025-03-23 14:24:11.264054885 +0100
@@ -77,12 +77,12 @@
 
 # See if spaces are valid in variable names (this was the case until
 # GNU Make 3.82)
-__gmsl spaced_var := $(true)
-ifeq ($(__gmsl spaced_var),$(true))
-__gmsl_spaced_vars := $(true)
-else
+# __gmsl spaced_var := $(true)
+# ifeq ($(__gmsl spaced_var),$(true))
+# __gmsl_spaced_vars := $(true)
+# else
 __gmsl_spaced_vars := $(false)
-endif
+# endif
 
 # Figure out whether we have $(eval) or not (GNU Make 3.80 and above)
 # if we do not then output a warning message, if we do then some
